RadioItem/RadioItem.js

```diff
@@ -91,16 +107,19 @@ const RadioItemBase = kind({
 	},
 
 	computed: {
-		className: ({css, selected, styler}) => styler.append(selected && css.selected)
+		className: ({css, inline, selected, styler}) => styler.append(selected && css.selected, {inline})
 	},
 
-	render: ({children, css, icon, selected, slotBefore, ...rest}) => {
+	render: ({children, css, icon, selected, skin, slotBefore, ...rest}) => {
+		const itemSize = skin === 'silicon' ? 'small' : 'large';
+
 		return (
 			<Item
 				aria-checked={selected}
 				role="checkbox"
 				{...rest}
 				css={css}
+				size={itemSize}
```

Comment:
Let's not make a constant here.
```suggestion
				size={skin === 'silicon' ? 'small' : 'large'}
```